demos/http/http_demo_plaintext/http_demo_plaintext.c

```diff
@@ -359,7 +330,7 @@ static int32_t transportRecv( HTTPNetworkContext_t * pContext,
 {
     int32_t bytesReceived = 0;
 
-    bytesReceived = ( int32_t ) recv( ( int ) pContext->tcpSocket, pBuffer, bytesToRecv, 0 );
+    bytesReceived = ( int32_t ) recv( pContext->tcpSocket, pBuffer, bytesToRecv, 0 );
```

Comment:
The `int32_t` cast is redundant as well

Reply:
`recv` and `send` return value is of type `ssize_t`. Is this automatically 32 bits? Not sure.

Reply:
As the variable being assigned to is `int32_t`, and explicit cast does not make a difference, unless the compiler is issuing a warning about it.

Reply:
Removed